src/website/app/demos/Dropdown/examples/dropdown.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2017 CA
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+import Button from '../../../../../Button';
+import IconCloud from '../../../../../Icon/IconCloud';
+import Dropdown from '../../../../../Dropdown';
+import data from '../../Menu/components/menuData';
+
+export default {
+  id: 'basic',
+  title: 'Dropdown',
```

Comment:
Could you make this "Example"? In the [component docs PR](https://github.com/mineral-ui/mineral-ui/pull/352) I made all the first examples titled "Example" and removed the Example H2 per request.

Applies to the menu examples too :)

Reply:
I'd prefer for you to make this update after this PR has been merged and during your rebase.   It is not just a text change, there is the h2, links pointing to the #examples anchor, etc that are more in the scope of your PR than this one.

Reply:
that's probably true. I'll include it with #346